### PR TITLE
fix temp dir

### PR DESCRIPTION
**Pull request by MXueguang**

### CODE REVIEW

File: tests/test_load_qrels.py [hunk 1]
```diff
@@ -241,6 +241,7 @@ def test_trec2018_bl(self):
 
     def test_trec2019_bl(self):
         qrels_path = search.get_qrels_file('trec2019-bl')
+        print(qrels_path)
```
Comment (lintool) [inline tests/test_load_qrels.py:244]: Probably don't need this debug in the final version?

**Review by lintool — APPROVED** [review]
Actually, it's not obvious to me the source of the issue... can you please explain?

**Comment by MXueguang** [comment]
the issue is because other tests sets custom PYSERINI_CACHE as temp_dir but didn't unset.
`test_load_topics_qrels` is suppose to use default `~/.cache` as PYSERINI_CACHE

**Comment by MXueguang** [comment]
fix https://github.com/castorini/pyserini/issues/615

**Pull request merged**